src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeModifiers.java

```diff
@@ -95,6 +95,7 @@ public static Tuple<GTRecipe, Integer> accurateParallel(MetaMachine machine, @No
         if (maxParallel == 1) {
             return new Tuple<>(recipe, 1);
         }
+        if(!(machine instanceof ITieredMachine))return new Tuple<>(recipe, 1);
```

Comment:
what if someone makes a primitive machine with a parallel hatch for some reason? (as an example) it still has to go through.

Reply:
What do you think should be done to solve it?